### PR TITLE
Add footerlink to Twinkle talk page

### PR DESCRIPTION
**Pull request by TheTVExpert**
Add a link to Twinkle talk page, so users can easily provide feedback on the script.

### CODE REVIEW

**Comment by Amorymeltzer** [comment]
Won't this leave some/all of them fairly cluttered?  I'm skeptical of the need for this, but maybe removing the doc link?  I've not checked them lately to see how useful they are, but I'd imagine more so that the talkpage (which, incidentally, the doc would lead indirectly to).

**Comment by siddharthvp** [comment]
I think this is a good idea. We could always do with more user feedback. ARV I think has the max number of footerlinks and this fill up the footer. Maybe increase the ARV dialog width by a tiny bit to better accommodate them all?

File: modules/friendlytalkback.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ Twinkle.talkback.callback = function() {
 	Window.addFooterLink('About {{talkback}}', 'Template:Talkback');
```
Comment (Amorymeltzer) [inline modules/friendlytalkback.js:33]: Unrelated, but this seems pointless?

**Comment by Amorymeltzer** [comment]
Extending ARV just for the footer seems weird, but I'm not sure there's a better way to incorporate an extra link.  Maybe making the guides to AIV/UAA/SPI just say "AIV info", etc.?  Or something similarly shorter?

File: modules/twinklearv.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ Twinkle.arv.callback = function (uid, isIP) {
 	Window.addFooterLink('About SPI', 'WP:SPI');
```
Comment (Amorymeltzer) [inline modules/twinklearv.js:35]: I think Wikipedia:Sockpuppet investigations/SPI/Guide to filing cases is probably a more useful link, in line with the others?

**Review by Amorymeltzer — APPROVED** [review]
Let's do it!  I'll make the changes I suggested locally, but otherwise all set!

**Comment by Amorymeltzer** [comment]
Merged as 1626040

**Pull request closed without merging**